src/fields/fp.rs

```diff
@@ -293,7 +293,7 @@ impl Fp {
     }
 
     /// Squares this element.
-    #[inline]
+    #[cfg_attr(inline, not(feature = "uninline-portable"))]
```

Comment:
These are all backwards; `inline` is being used as the condition predicate, and `not(feature = "uninline-portable")` the applied attribute. I am surprised it compiles at all, let alone runs with the desired effect and passes tests.

Reply:
facepalm . But I'm not surprised it decreased size of the binary, because it definitely removes the `inline` attribute.

Reply:
fixed: 52af8293a8e31d27ae0cb712505cf6daf2f36549